src/chains/ethereum/transaction/src/transaction-receipt.ts

```diff
@@ -153,7 +153,7 @@ export class TransactionReceipt {
     const transactionHash = transaction.hash;
     const transactionIndex = transaction.index;
     blockLog.blockNumber = blockNumber;
-    raw[3].forEach(l => blockLog.append(transactionIndex, transactionHash, l));
+    raw[3].forEach(l => blockLog.append(blockLog[0], transactionIndex, transactionHash, l));
```

Comment:
I think it is. Honestly I can't remember why I wrote it this way instead of just passing `false`.

Passing booleans as positional args kinda makes me squirm from a readability standpoint, though. It makes me inclined to suggest that `append` should be refactored such that `removed` is an optional argument that defaults to `false`, but then that would make the argument ordering inconsistent with the ordering of the values in `BlockLog[_raw][1]`... 🤔🤔